builder/fuses.py

```diff
@@ -469,6 +482,11 @@ def get_lock_bits(target):
     jtagen = board.get("hardware.jtagen", "no").lower()
     ckout = board.get("hardware.ckout", "no").lower()
     cfd = board.get("hardware.cfd", "no").lower()
+    bootloader_type = board.get("bootloader.type", "urboot").lower()
+    if "no_bootloader" in (uart, bootloader_type):
+        uart = bootloader_type = "no_bootloader"
+    if "optiboot" in bootloader_type:
+        bootloader_type = "optiboot"
```

Comment:
This piece of code just reassigns the same value, is it intentional?

Reply:
I did this just in case someone called their bootloader `optiboot_flash`, which is what I've called the Optiboot fork I've been bundling for years.